packages/template-retail-react-app/app/components/link/index.jsx

```diff
@@ -19,7 +19,7 @@ const Link = React.forwardRef(({href, to, useNavLink = false, ...props}, ref) =>
             as={useNavLink ? NavSPALink : SPALink}
             {...(useNavLink && {exact: true})}
             {...props}
-            to={_href === '/' ? '/' : updatedHref}
```

Comment:
With multi-site implementation in place, the condition to keep '/' as is out of date. When the href is a '/', it should also be passed to `buildUrl` to generate the correct homepage url according to current site and locale